tests/dotnet/UnitTests/ApplePlatformExtensions.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+
+namespace Xamarin.Utils {
+	public static class ApplePlatformExtensionsWithVersions {
+		public static string ToFrameworkWithDefaultVersion (this ApplePlatform @this)
```

Comment:
This is a little confusing to me...
From quick googling, it looks like the '@this' allows you to use '@this' as a variable name, but why do we want to do this here?

Reply:
This is an [extension method](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/extension-methods), which allows you to write methods that kind of work like instance methods. In instance methods you use `this` to refer to the instance, and while you can't use `this` in an extension method, you can use `@this`(the `@` sign makes it so that `this` isn't a keyword, just a normal variable name), and that makes it look more like an instance method. In other words: it's just a convention to make extension methods look more like instance methods.